docs/usage.md

```diff
@@ -147,6 +147,12 @@ $(function() {
 		<td valign="top"><code>int</code></td>
 		<td valign="top"><code>60</code></td>
 	</tr>
```

Comment:
see comment in src/deafaults.js
If that is updated this would need updated too

Reply:
Noted.